### PR TITLE
volunteer form submission confirmation

### PR DESCRIPTION
**Pull request by willblake01**
Add a confirmation page showing volunteer registration was successful or not. Prevents ambiguity for users and should prevent contacting admins directly.

-Add show to /volunteers view
-Update controller with event_volunteer_path

[finishes #175542908]

### CODE REVIEW

**Comment by willblake01** [comment]
image

**Comment by willblake01** [comment]
Sad path image:
image

File: app/controllers/volunteers_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def create
     @volunteer = Volunteer.new(volunteer_params)
     result = CreateVolunteer.new(volunteer: @volunteer).call
     if result.success
-      redirect_to(root_path, success: result.message)
+      redirect_to(event_volunteer_path(id: result.object.id), success: result.message)
```
Comment (ColinW520) [inline app/controllers/volunteers_controller.rb:12]: this can just be `event_volunteer_path(event, result.object)`

**Review by ColinW520 — APPROVED** [review]
🚀 as long as that redirect is working as expected

**Pull request merged**